packages/site/src/components/PaymentChannelDetails.tsx

```diff
@@ -1,7 +1,172 @@
-type Props = {
-  paymentChannel: string;
-};
+import {
+  Box,
+  LinearProgress,
+  Typography,
+  Stack,
+  Button,
+  SvgIcon,
+} from "@mui/material";
+import { FC, useEffect, useMemo, useState } from "react";
+import { makeStyles } from "tss-react/mui";
+import { PhoneArrowUpRightIcon, UserIcon } from "@heroicons/react/24/outline";
 
-export default function PaymentChannelDetails({ paymentChannel }: Props) {
-  return <p>{paymentChannel}</p>;
+interface PaymentChannelDetails {
+  channelID: string;
+  counterparty: string; // address for now - can abstract to something richer later
+  capacity: number; // total value locked in channel
```

Comment:
I think these need to be `bigint`s and not `number`. Since these balances are denominated in `wei` they can get quite large and overflow a `number`

Reply:
Okay